tests/manage/z_cluster/cluster_expansion/test_vm_delete_and_replace.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    tier4a,
+    E2ETest,
+    vsphere_platform_required,
+    skipif_vsphere_ipi,
+    bugzilla,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.utility import vsphere
+from ocs_ci.ocs.node import verify_all_nodes_created, get_osd_running_nodes
```

Comment:
change to alphabetical order?

Also it's good to have alphabetically ordered all imports.
So:

```
#First
from ocs_ci.framework import config
from ocs_ci.framework.testlib import (...
from ocs_ci.helpers.sanity_helpers import
from ocs_ci.ocs.resources import pod
from ocs_ci.utility import vsphere
from ocs_ci.ocs.node import ...
```

Hope you got my point. Order should be in all imports as well as multiple imports from one module.

Reply:
Done